### DISCUSSION TITLE
Uploading large image fails, no error is shown

### DISCUSSION THREAD

**Discussion by nzyeehaa** [datenstrom/community#479] [Report a bug] [answered]
when uploading an image, if it is over _weight,_ nothing happens and no error is shown leaving the user clueless.

I refer to image **weight** as being the amount of kb space required to store it. 
I refer to image **size** as being dimensions which image manager does a great job with.

I generally aim under 100kb for website images - so load speed isnt affected as much. 
But if a user tries to upload a 1.2mb image - no error is given.

I cant find any setting except _ImageUploadWidthMax: 640_
no error in yellow.log
and nothing on screen

is that image weight only set by php?

**Comment by markseuffert** [comment, marked as answer]
Thank you for reporting this bug. 

File upload to the web server is limited by `upload_max_filesize` in your `php.ini` file. It looks like files with a larger size were completely ignored on the client side, so that explains why there was no error message. Should be fixed now.

**Comment by nzyeehaa** [comment]
Hi @markseu,
thanks for your speedy response.
following update, still no error message here

**Comment by markseuffert** [comment]
Can you test uploading a markdown file, do you get an "file format not supported" error message?

**Comment by nzyeehaa** [comment]
hi @markseu 

upload .md file gives error.
upload large image file now gives error also - works as intended

i did a hard refresh after updating, got no errors from large files, but seems to be working fine now.
thanks
